package.json

```diff
@@ -36,12 +38,16 @@
     "history": "2.0.0-rc2",
     "immutable": "3.7.6",
     "lz-string": "1.4.4",
+    "moment": "2.11.1",
     "node-uuid": "1.4.7",
     "object-assign": "4.0.1",
     "raf": "3.1.0",
+    "rd3": "^0.7.0",
     "react": "0.14.7",
     "react-copy-to-clipboard": "3.0.4",
-    "react-date-picker": "4.0.10",
+    "react-d3-components": "^0.6.4",
```

Comment:
Can you pin the dependencies to a specific version just like the others? Thanks.